### DISCUSSION TITLE
Where to find the jar file

### DISCUSSION THREAD

**Discussion by prasathsekar** [apache/pulsar#22462] [Q&A] [answered]
There is no Jar file found for the below connector

https://pulsar.apache.org/docs/next/io-mongo-source/

**Comment by alpreu** [comment]
Hi @prasathsekar, website description is wrong unfortunately. It is the same .nar as the mongo sink connector:

https://pulsar.apache.org/download/#connectors:~:text=asc%2Csha512-,MongoDB%20sink,-pulsar%2Dio%2Dmongo

**Reply by prasathsekar** [reply]
We need to use pulsar-io-mongo-3.2.2.nar for both source and sink ?

https://www.apache.org/dyn/closer.lua/pulsar/pulsar-3.2.2/connectors/pulsar-io-mongo-3.2.2.nar?action=download

**Reply by alpreu** [reply, marked as answer]
> We need to use pulsar-io-mongo-3.2.2.nar for both source and sink ?

Exactly, the NAR contains the code for both source and sink. 
You can specify the [sink/source class](https://github.com/apache/pulsar/blob/branch-3.2/pulsar-io/mongo/src/main/resources/META-INF/services/pulsar-io.yaml#L21-L24) when starting the connector via cli/api
